gyp/pylib/gyp/simple_copy.py

```diff
@@ -28,8 +28,14 @@ def deepcopy(x):
 def _deepcopy_atomic(x):
   return x
 
-for x in (type(None), int, long, float,
-          bool, str, unicode, type):
+import sys
+if sys.version_info < (3, 0):
```

Comment:
On this one, can we please follow Python porting best practice [__use feature detection instead of version detection__](https://docs.python.org/3/howto/pyporting.html#use-feature-detection-instead-of-version-detection)?

Reply:
extracted checks to `compat` module